Makefile

```diff
@@ -13,63 +13,17 @@ bootstrap:
 bootstrap-oss:
 	@echo "  ⚠️  Bootstrapping open-source Point-Free..."
 	@set -e; set -o pipefail; $(MAKE) .env | sed "s/make\[1\]: \`\.env'/\  ✅ $$(tput bold).env$$(tput sgr0)/"
-	@$(MAKE) xcodeproj-oss
-	@$(MAKE) install-mm
+	@$(MAKE) check-dependencies
 	@echo "  ✅ Bootstrapped! Opening Xcode..."
-	@sleep 1 && xed .
-
-bootstrap-oss-lite:
-	@echo "  ⚠️  Bootstrapping open-source Point-Free (lite)..."
-	@$(MAKE) xcodeproj-oss
-	@echo "  ✅ Bootstrapped! Opening Xcode..."
-	@sleep 1 && xed .
+	@xed .
 
 bootstrap-private:
 	@echo "  👀 Bootstrapping Point-Free (private)..."
-	@$(MAKE) xcodeproj
-	@$(MAKE) install-mm
+	@$(MAKE) check-dependencies
 	@echo "  ✅ Bootstrapped! Opening Xcode..."
 	@sleep 1 && xed .
 
-uninstall: uninstall-mm db-drop
-
-install-mm:
```

Comment:
I think we lose playground support...but Xcode previews should work even better for our purposes (when we're 5.1-compatible).